### PR TITLE
Snowflake Fixes and Updates

### PR DESCRIPTION
**Pull request by EmilyNecciai**
# Pull Request/Issue Resolution

**Description of Change:**
- Snowflake syntax fixes in examples
- additional example in S3 > Snowflake

---

### CODE REVIEW

File: _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Then, in Snowflake, create a Snowflake S3 Stage (called `braze_data`) as followi
 create or replace stage currents.public.braze_data
     url='s3://snowpipe-demo/'
     credentials = (AWS_KEY_ID = '...' AWS_SECRET_KEY = '...' );
-show stages;
+show pipes;
```
Comment (zzhaobraze) [inline _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md]: show stages; should be correct if this is about stages

File: _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ create file format currents.public.currents_avro type = 'avro' compression = 'au
 alter stage currents.public.braze_data set file_format = currents.public.currents_avro;
 ```
 
-Finally, use the `show stages;` command to show your SQS information. The name of the SQS queue will be visible in a new column called `NOTIFICATION_CHANNEL`.
```
Comment (zzhaobraze) [inline _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md]: show stages should be correct if this is about stages

File: _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/amazon_s3.md [hunk 3]
```diff
@@ -6,38 +6,38 @@ page_order: 1
 
 # Amazon S3 Integration
 
-[Amazon S3](https://aws.amazon.com/s3/) is a highly scalable storage system offered by Amazon Web Services. Braze has two different integration strategies with Amazon S3 - one for [Braze Currents]({{ site.baseurl }}/partners/braze_currents/about/) and one for all Dashboard data exports (CSV exports, Engagement Reports, etc.) Both integrations support two different authentication/authorization methods:
+[Amazon S3](https://aws.amazon.com/s3/) is a highly scalable storage system offered by Amazon Web Services. Braze has two different integration strategies with Amazon S3 - one for [Braze Currents](<{{ site.baseurl }}/partners/braze_currents/about/>) and one for all Dashboard data exports (CSV exports, Engagement Reports, etc.) Both integrations support two different authentication/authorization methods:
```
Comment (zzhaobraze) [inline _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/amazon_s3.md]: why was the < and > added?

Comment (EmilyNecciai) [reply _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/amazon_s3.md]: That's weird

File: _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md [hunk 4]
```diff
@@ -31,24 +31,24 @@ In AWS, create a new __public-private key pair__ with grants according to your o
 
 Then, in Snowflake, create a Snowflake S3 Stage (called `braze_data`) as followis:
```
Comment (klockett7) [inline _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md]: should be 'follows' instead of 'followis'

**Comment by zzhaobraze** [comment]
@cla-bot check

File: _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md [hunk 5]
```diff
@@ -133,7 +133,7 @@ You must repeat the CREATE TABLE and CREATE PIPE commands for every event type.
 
 First, create a table INTO which we will continuously load using the following structure:
 
-```js
+```sql
 CREATE TABLE "CURRENTS_PM"."PUBLIC".PUBLIC_USERS_MESSAGES_PUSHNOTIFICATION_OPEN (
```
Comment (lenny-braze) [inline _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md]: please remove "CURRENTS_PM"."PUBLIC". from all queries

File: _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md [hunk 6]
```diff
@@ -163,11 +163,11 @@ Then, create the AUTO continuous load pipe and specify
 1. which table to load and
 2. how to load the following table.
 
-```js
+```sql
 CREATE OR REPLACE PIPE PIPE_USERS_MESSAGES_PUSHNOTIFICATION_OPEN
-                            auto_ingest=true
+                            auto_ingest=true AS
 COPY INTO "CURRENTS_PM"."PUBLIC".USERS_MESSAGES_PUSHNOTIFICATION_OPEN
```
Comment (lenny-braze) [inline _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md]: please see above

File: _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md (hunk 1 above)
Comment (lenny-braze) [reply _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md]: agreed

File: _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md (hunk 2 above)
Comment (lenny-braze) [reply _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md]: I think it's worth while mentioning that 'show stages' or 'show pipes' both display notification channel

Comment (EmilyNecciai) [reply _docs/_partners/braze_currents/advanced_topics/s3_to_snowflake.md]: So which should I do? Stages or pipes?  @lenny-braze

File: _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md [hunk 7]
```diff
@@ -123,6 +123,93 @@ ORDER BY 1
...
+$1:canvas_id::STRING,
+$1:canvas_name::STRING,
+$1:canvas_variation_id::STRING,
+$1:canvas_step_id::STRING,
+$1:send_id::STRING,
+$1:dispatch_id::STRING,
+$1:email_address::STRING,
+$1:url::STRING,
+$1:sending_ip::STRING,
+$1:user_agent::STRING from @braze_currents.public.braze_data/currents/dataexport.prod-03.S3.integration.ZZZZZZZZZ_YOUR_INTEGRATION_ID_HERE/event_type=users.messages.email.Click/);
```
Comment (lenny-braze) [inline _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md]: what is ZZZZZZZ?

Comment (EmilyNecciai) [reply _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md]: I'm sure this is some kind of filler for the ID. I copied this from the resources I was given.

Comment (EmilyNecciai) [reply _docs/_partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake.md]: If there would be a better filler, let me know, I can change it.

**Comment by EmilyNecciai** [comment]
@lenny-braze I'll make these changes, but you need to change your review to "approved" to unblock the merge. 
Thanks!

**Comment by EmilyNecciai** [comment]
@cla-bot check

**Pull request merged**